<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmKrajDana</class>
 <widget class="QDialog" name="frmKrajDana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Stanje kase</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Datum od</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="6">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="txtPologKasa">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Datum do</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="radioStanjeVrijeme">
         <property name="text">
          <string>Vrijeme</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Prikazi</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Polog</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="btnUzimanjeGotovine">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Uzimanje gotovine iz blagajne</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Prikazi</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btnPrikaziDatum">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Dat</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="btnIspisStanjeKase">
         <property name="text">
          <string>Ispis POS</string>
         </property>
         <property name="icon">
          <iconset resource="../kasares.qrc">
           <normaloff>:/ikone/ikone/document_print.svg</normaloff>:/ikone/ikone/document_print.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="btnIspisStanjeKaseVeliki">
         <property name="text">
          <string>Ispis Veliki</string>
         </property>
         <property name="icon">
          <iconset resource="../kasares.qrc">
           <normaloff>:/ikone/ikone/document_print.svg</normaloff>:/ikone/ikone/document_print.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="btnIspisStanjeKase_2">
         <property name="text">
          <string>Detaljan PDF</string>
         </property>
         <property name="icon">
          <iconset resource="../kasares.qrc">
           <normaloff>:/ikone/ikone/document_print.svg</normaloff>:/ikone/ikone/document_print.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="txtStanjeDatumOd">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>0</height>
          </size>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2014</year>
           <month>11</month>
           <day>24</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="txtStanjeDatumDo">
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2014</year>
           <month>11</month>
           <day>24</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QTimeEdit" name="txtStanjeSatOdN">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::HourSection</enum>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QTimeEdit" name="txtStanjeSatDoN">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Zakljucak smjene korisnik</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QTableView" name="tableZakljucakKorisnik"/>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Datum od</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Datum do</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="radioZakljKvrijeme">
           <property name="text">
            <string>Vrijeme</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Prikazi</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="txtZakljKDatumOd">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>0</height>
            </size>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QTimeEdit" name="txtZakljKSatOd"/>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="txtZakljKDatumDo">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>0</height>
            </size>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QTimeEdit" name="txtZakljKSatDo"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnIspisZakljucakKorisnik">
         <property name="text">
          <string>Ispis</string>
         </property>
         <property name="icon">
          <iconset resource="../kasares.qrc">
           <normaloff>:/ikone/ikone/document_print.svg</normaloff>:/ikone/ikone/document_print.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../kasares.qrc"/>
 </resources>
 <connections/>
</ui>
